webapp/src/main/webapp/js/individual/propertyGroupControls.js

```diff
@@ -163,27 +134,23 @@ $(document).ready(function(){
             // So ensure that the tab in local storage has been rendered on the page.
             if ( $("ul.propertyTabsList li[groupName='" + groupName + "']").length ) {
                 // if the selected tab is the default first one, don't do anything
-                if ( $('li.clickable').first().attr("groupName") != groupName ) {
+                if ( $('li.nav-link').first().attr("groupName") != groupName ) {
                     // deselect the default first tab
-                    var $firstTab = $('li.clickable').first();
-                    $firstTab.removeClass("selectedGroupTab clickable");
-                    $firstTab.addClass("nonSelectedGroupTab clickable");
-                    // select the stored tab
-                    $("li[groupName='" + groupName + "']").removeClass("nonSelectedGroupTab clickable");
-                    $("li[groupName='" + groupName + "']").addClass("selectedGroupTab clickable");
-                    // hide the first tab section
-                    $('section.property-group:visible').hide();
 
                     if ( groupName == "viewAll" ) {
-                        processViewAllTab();
-                    }
+                        console.log("View all")
```

Comment:
suggest removing console debugging print

Reply:
Thanks, I removed it.